apps/web/src/components/SendFlow/FormPageHeader.tsx

```diff
@@ -1,31 +1,14 @@
-import { Heading, ModalCloseButton, ModalHeader, Text, chakra } from "@chakra-ui/react";
-
-import { useColor } from "../../styles/useColor";
-
-export const HeaderWrapper = chakra(ModalHeader, {
-  baseStyle: {
-    padding: 0,
-    paddingBottom: "32px",
-    textAlign: "center",
-  },
-});
+import { Heading, ModalCloseButton, ModalHeader } from "@chakra-ui/react";
 
 export const FormPageHeader = ({
   title = "Send",
-  subTitle = "Send one or insert into batch",
+  subTitle: _subTitle,
```

Comment:
what is the purpose of `subTitle` prop here since it's not used in this component?

Reply:
it's just to not update it altogether for all other modal windows, will be done later when we migrate all of the forms